spec/features/admin/home_redirect_spec.rb

```diff
@@ -0,0 +1,52 @@
+require "rails_helper"
+
+feature "home redirect" do
+  let(:actor) { create(:admin) }
+
+  around { |ex| with_user_home_subdomain(actor) { ex.run } }
+
+  before do
+    login_as(actor, scope: :user)
+  end
+
+  scenario "admin changes default home and visits root url" do
+    visit "/"
+    expect(page).to have_title("Directory")
+
+    ["Meals", "Directory", "Reservations", "Wiki"].each { |new_default|
+      change_default_home(new_default)
+      visit "/"
+      expect(page).to have_title(new_default)
+    }
+  end
+end
+
+def change_default_home(new_default)
+    visit "/admin/settings/community"
+    select(new_default, from: "Default landing page")
```

Comment:
Field labels should always be Title Case. You'll see this when you add the translation.